rpcs3/Emu/RSX/GL/GLPresent.cpp

```diff
@@ -269,7 +269,7 @@ void GLGSRender::flip(const rsx::display_flip_info_t& info)
 		// TODO: Implement FSR for OpenGL and remove this fallback to bilinear
 		const gl::filter filter = g_cfg.video.output_scaling == output_scaling_mode::nearest ? gl::filter::nearest : gl::filter::linear;
 
-		if (use_full_rgb_range_output && rsx::fcmp(avconfig.gamma, 1.f) && !avconfig._3d)
+		if (use_full_rgb_range_output && rsx::fcmp(avconfig.gamma, 1.f) && avconfig.stereo_mode != stereo_render_mode_options::disabled)
```

Comment:
I think this is a typo
```suggestion
		if (use_full_rgb_range_output && rsx::fcmp(avconfig.gamma, 1.f) && avconfig.stereo_mode == stereo_render_mode_options::disabled)
```